tasks/configure.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Get Current Global Python Version
+  become: yes
+  become_user: "{{ sansible_pyenv.user }}"
+  command: bash -lc "pyenv global"
+  changed_when: no
+  register: global_version
+
+- name: Activate Python Version
+  become: yes
+  become_user: "{{ sansible_pyenv.user }}"
+  command: "bash -lc 'pyenv global {{ sansible_pyenv.python_version }}'"
```

Comment:
The content of variable sansible_pyenv.python_version is not coupled between the dev and ops halves (ie build and configure stages) of the deployment process..
What happens if a python_version that was not installed in the build stage is specified here?

Reply:
It would fail.  I could ensure the version is installed right at the beginning of configure.yml, if you think that would be nicer.

Reply:
I am failing now at the beginning if the requested Python version is not installed.

Reply:
Personally I think that all of this should probably reside in build.yml, the configure stage is really meant for configuring machines when they boot up so they can accept environment variables, I don't think we would ever want to install pyenv and then configure it on boot.

Reply:
Moved to build.yml